pkg/kubelet/kubelet.go

```diff
@@ -2892,6 +2892,58 @@ func (kl *Kubelet) canResizePod(pod *v1.Pod) (bool, v1.PodResizeStatus, string)
 	return true, v1.PodResizeStatusInProgress, ""
 }
 
+func IsMemoryResize(currentPod, allocatedPod *v1.Pod) bool {
```

Comment:
nit: we've been using the term "desired resources" to refer to the requested resize resources before they're allocated, so I'd prefer to call this `desiredPod`
```suggestion
func IsMemoryResize(desiredPod, allocatedPod *v1.Pod) bool {
```